.github/workflows/dart.yml

```
@@ -88,34 +88,73 @@
+      - uses: actions/checkout@v3
+
+      - name: Download coverage reports
+        uses: actions/download-artifact@v3
+        with:
+          path: coverage-artifacts
+
+      - name: Install lcov
+        run: sudo apt-get install -y lcov
+
+      - name: Merge coverage reports
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2038:warning:2:1: Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames

(shellcheck)